ra/ra.go

```diff
@@ -640,6 +641,12 @@ func (ra *RegistrationAuthorityImpl) NewCertificate(ctx context.Context, req cor
 		return emptyCert, err
 	}
 
+	if ra.Publisher != nil {
+		go func() {
+			_ = ra.Publisher.SubmitToCT(context.Background(), cert.DER)
```

Comment:
In the CA version of this code we have a comment explaining the use of `context.Background()`. Do you think its worth including that here as well?